test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -177,6 +177,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'telemetry.labels.ciBuildId (string)',
         'telemetry.labels.ciBuildJobId (string)',
         'telemetry.labels.ciBuildNumber (number)',
+        'telemetry.labels.environment (string)',
```

Comment:
What sort of values can we expect to be exposed here? Will they contain sensitive information?